tests/commerce_integration/TEC/Tickets/Commerce/Partials/Checkout/Cart/__snapshots__/CartFooterTest__test_should_render_cart__0.snapshot.html

```diff
@@ -1,9 +1,12 @@
-<footer  class="tribe-tickets__commerce-checkout-cart-footer tribe-common-b1" >
 
-	<div class="tribe-tickets__commerce-checkout-cart-footer-quantity">
+<footer  class="tribe-tickets__commerce-checkout-cart-footer tribe-common-b1" >
+	
+<div class="tribe-tickets__commerce-checkout-cart-footer-order-modifier-fees">
+	<ul>
+			</ul>
+</div>
```

Comment:
Maybe that this is here is an indication that something isn't going as expected ? Since there no fees should it be printed at all ?